v1.1/uuid.md

```diff
@@ -0,0 +1,135 @@
+---
+title: UUID
+summary: The UUID data type .
+toc: false
+---
+
+The `UUID` (Universally Unique Identifier) [data type](data-types.html) stores a 128-bit value that is [unique across both space and time](https://www.ietf.org/rfc/rfc4122.txt). 
+
+<div id="toc"></div>
+
+## Syntax
+`UUID` can be expressed using the following formats:
+
+Format | Description 
+-------|-------------
+Standard [RCF4122](http://www.ietf.org/rfc/rfc4122.txt)-specified format. | Hyphen-seperated groups of 8, 4, 4, 4, 12 hexadecimal digits<br><br> Example: acde070d-8c4c-4f0d-9d8a-162843c10333
+With braces | The standard [RCF4122](http://www.ietf.org/rfc/rfc4122.txt)-specified format with braces.<br><br>Example: {acde070d-8c4c-4f0d-9d8a-162843c10333}
+As Bytes | UUID value specified as bytes.<br><br>Example: b'kafef00ddeadbeed'
+UUID used as a URN | UUID can be used as a Uniform Resource Name (URN). In that case, the format is [specified](https://www.ietf.org/rfc/rfc2141.txt) as "urn:uuid:" followed by standard [RCF4122](http://www.ietf.org/rfc/rfc4122.txt)-specified format.<br><br>Example: 'urn:uuid:63616665-6630-3064-6465-616462656564'
+
+## Size
+A `UUID` column is 128 bits in width, but the total storage size is likely to be larger due to CockroachDB metadata. 
+
+## Examples
+
+### Create a table with manually-entered UUID values
```

Comment:
It's not really the norm, but this certainly doesn't hurt. So also fine to leave as-is!